doc/classes/Environment.xml

```diff
@@ -425,10 +425,12 @@
 		</constant>
 		<constant name="TONE_MAPPER_FILMIC" value="2" enum="ToneMapper">
 			Uses a film-like tonemapping curve to prevent clipping of bright values and provide better contrast than [constant TONE_MAPPER_REINHARDT]. Slightly slower than [constant TONE_MAPPER_REINHARDT].
+			[b]Note:[/b] This tonemapper does not support HDR output and clips output to the [code]0.0-1.0[/code] range. A different tonemapper should be used when rendering to an HDR display.
```

Comment:
There's many ways ranges are expressed in the docs but this is not one of them. Although I am more of a fan of saying something like "from `0.0` to `1.0`", there's also an overwhelming amount of the following format in the "rendering" side of things.
```suggestion
			[b]Note:[/b] This tonemapper does not support HDR output because it clips output in the [code][0.0, 1.0][/code] range. It is recommended to use a different tonemapper when rendering to an HDR display.
```